### ISSUE TITLE
I cannot reproduce the the same results in predict_ogt.ipynb

### ISSUE THREAD

**Issue by zhyhan** [ai4protein/Pro-Prime#4]
Dear authors,

Thanks for the contributions. I tried to reproduce the same results in predict_ogt.ipynb, however, I obtained a difference results as follows:

[23.50634765625, 27.181840896606445, 26.907215118408203, 17.608062744140625, 25.18841552734375, 28.461463928222656, 27.530790328979492, 27.2039852142334, 20.377643585205078, 20.891084671020508, 19.0797176361084, 17.098712921142578, 20.04912757873535, 26.564207077026367]

**Comment by ginnm** [comment]
Please carefully check your code. 
<img width="374" alt="image" src="https://github.com/user-attachments/assets/163f0216-0ae2-44f6-ae6b-d12782e4121c">

**Comment by ginnm** [comment]
If you meet this problem again, pleas offer a copy of your code, thanks.

**Comment by zhyhan** [comment]
Thanks for your timely repoly. The code is below:

from transformers import AutoTokenizer, AutoModel
import torch
import pandas as pd
from Bio import SeqIO
from tqdm import tqdm

sequence_file = "notebooks/example.fasta"

model_path = "AI4Protein/Prime_690M"
device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
model = AutoModel.from_pretrained(model_path, trust_remote_code=True)
tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
model.eval()
model = model.to(device)

togt = []
with torch.no_grad():
    for record in tqdm(list(SeqIO.parse(sequence_file, "fasta"))):
        sequence = str(record.seq)
        tokenied_results = tokenizer(sequence, return_tensors="pt")
        input_ids = tokenied_results.input_ids.to(device)
        attention_mask = tokenied_results.attention_mask.to(device)
        logits = model(input_ids=input_ids, attention_mask=attention_mask).predicted_values
        togt.append(logits.item())

print(togt)

**Comment by ginnm** [comment]
PyTorch Version?

**Comment by ginnm** [comment]
<img width="1162" alt="image" src="https://github.com/user-attachments/assets/de14db39-c872-4a82-b821-4f75c6bd1704">

**Comment by ginnm** [comment]
Or you should add "force_download=True"

**Comment by zhyhan** [comment]
pytorch version is 2.0.1.

**Comment by ginnm** [comment]
🤔 That is really strange. Can you try it on another computer?

**Comment by AquaporinCN** [comment]
@zhyhan I can reproduce the same result, with pytorch version as 2.5.
